packages/admin/src/pages/api/unblockUser.page.ts

```diff
@@ -0,0 +1,17 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { updateUserRoles } from '../../services/SuperAdminService';
+import { getUserTokenFromCookies } from '../../utils/session';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const id = req.query.id as string;
+  const jwt = getUserTokenFromCookies(req);
+  const newUserRoles = ['1', '2'];
```

Comment:
should probably have these in a constants or something

Reply:
Will assign them to find and applicant to be a bit more readable